Mathlib/Analysis/Normed/Module/Dual.lean

```diff
@@ -159,6 +159,14 @@ variable {E : Type*} [SeminormedAddCommGroup E] [NormedSpace 𝕜 E]
 theorem mem_polar_iff {x' : Dual 𝕜 E} (s : Set E) : x' ∈ polar 𝕜 s ↔ ∀ z ∈ s, ‖x' z‖ ≤ 1 :=
   Iff.rfl
 
+@[simp]
+theorem zero_mem_polar (s : Set E) : (0 : Dual 𝕜 E) ∈ polar 𝕜 s := fun _ _ => by
```

Comment:
Why not reuse `LinearMap.zero_mem_polar` in the proof? I think that `LinearMap.zero_mem_polar _ s` will work as a proof.

Reply:
Yes, done, thanks.